cmd/ejson/main.go

```diff
@@ -16,6 +17,8 @@ func execManpage(sec, page string) {
 }
 
 func main() {
+	// Encryption is expensive. We'd rather burn cycles on many cores than wait.
+	runtime.GOMAXPROCS(runtime.NumCPU())
```

Comment:
It might be nice to make the default use all cores and allow the command line to override with another value. But whatever.